files/tarball_ingester_minio.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+MINIO_ACCESS_KEY=$(aws secretsmanager get-secret-value --secret-id minio --query SecretString --output text | jq -r .MINIO_ACCESS_KEY)
+MINIO_SECRET_KEY=$(aws secretsmanager get-secret-value --secret-id minio --query SecretString --output text | jq -r .MINIO_SECRET_KEY)
+S3_MINIO_BINARY="s3://${1}/minio/minio.RELEASE.2020-10-28T08-16-50Z"
+
+echo "Downloading min.io binary..."
+aws s3 cp "$S3_MINIO_BINARY" /usr/bin/minio
+echo "Changing permissions for MinIO"
+chmod +x /usr/bin/minio
+minio --version
+
+echo "Creating user minio"
+useradd minio -m
```

Comment:
To check; does this create a no-shell user (see https://github.com/minio/minio-service#guidelines).

Reply:
was copy-pasted and should in fact be a capital M, an no that's not no-shell, replaced with:

useradd minio -M --shell=/bin/false